src/en/ref/Database Mapping/column.adoc

```diff
@@ -84,7 +84,7 @@ import groovy.transform.CompileStatic
 @CompileStatic
 class BootStrap {
 
-    def init = { servletContext ->
+    def init = {
         Book.saveAll(   
```

Comment:
Will this actually work without a `@Transactional` or wrapping in a `Book.withNewSession` ?